app/client/src/components/pages/Actions.js

```diff
@@ -720,7 +720,18 @@ function Actions() {
                                   }
                                   subTitle={
                                     <>
-                                      {orgLabel} {auId}
+                                      {orgLabel} {auId}{' '}
+                                      {!graphic && (
+                                        <>
+                                          <br />
+                                          <span css={noMapDataWarningStyles}>
+                                            <i className="fas fa-exclamation-triangle" />
+                                            <strong>
+                                              [Waterbody not visible on map.]
+                                            </strong>
+                                          </span>
+                                        </>
+                                      )}
```

Comment:
The warning flickers for all waterbodies when the page first loads, since the map is loading. While it is technically true that the waterbody isn't visible on the map while the map is loading, I think it would be better to wait until the `mapLayer` settles:
```suggestion
                                      {mapLayer.status === 'success' && !graphic && (
                                        <>
                                          <br />
                                          <span css={noMapDataWarningStyles}>
                                            <i className="fas fa-exclamation-triangle" />
                                            <strong>
                                              [Waterbody not visible on map.]
                                            </strong>
                                          </span>
                                        </>
                                      )}
```

I'd say the warning is unnecessary if the map layer is in the 'failure' state, but the condition could be revised to `mapLayer.status !== 'fetching'`.

Reply:
Good catch!